<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hash tables</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QStackedWidget" name="m_stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="m_firstPage">
       <layout class="QVBoxLayout" name="firstPageLayout">
        <item>
         <widget class="QTabWidget" name="m_tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="TextBrowserWidget" name="m_theoryTab">
           <attribute name="title">
            <string>Theory</string>
           </attribute>
          </widget>
          <widget class="DemonstrationWidget" name="m_demonstrationTab">
           <attribute name="title">
            <string>Demonstration</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="TestingWidget" name="m_secondPage">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomLayout">
      <item>
       <spacer name="bottomSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="m_exitButton">
        <property name="text">
         <string>&amp;Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1093</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="m_actionManual"/>
    <addaction name="separator"/>
    <addaction name="m_actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="m_actionTheory"/>
    <addaction name="m_actionPractice"/>
   </widget>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="m_statusbar"/>
  <action name="m_actionNewUserSession">
   <property name="text">
    <string>&amp;User session</string>
   </property>
   <property name="toolTip">
    <string>User session</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="m_actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="m_actionSave">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="m_actionSaveAs">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="m_actionOptions">
   <property name="text">
    <string>&amp;Options...</string>
   </property>
  </action>
  <action name="m_actionManual">
   <property name="text">
    <string>&amp;Manual</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="m_actionAbout">
   <property name="text">
    <string>About this &amp;program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="m_actionTheory">
   <property name="text">
    <string>&amp;Theory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="m_actionPractice">
   <property name="text">
    <string>&amp;Practice</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="m_actionAllSessions">
   <property name="text">
    <string>&amp;General</string>
   </property>
  </action>
  <action name="m_actionCurrentSessions">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Current</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TestingWidget</class>
   <extends>QWidget</extends>
   <header>testingWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DemonstrationWidget</class>
   <extends>QWidget</extends>
   <header>demonstrationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TextBrowserWidget</class>
   <extends>QWidget</extends>
   <header>textBrowserWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_tabWidget</tabstop>
  <tabstop>m_exitButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1044</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>935</x>
     <y>524</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
